sktime/classification/distance_based/tests/test_time_series_neighbors.py

```diff
@@ -28,8 +28,8 @@
 def test_knn_on_unit_test():
```

Comment:
this will likely clash with the refactor of @MatthewMiddlehurst - this is now part of the classifier test class. Should be moved there and/or uniformized.

Reply:
we can resolve any clashes once one of the PRs are in, it will be simple